src/resources/postcss/views/month/_multiday-event.pcss

```diff
@@ -42,4 +75,42 @@
 	.tribe-events-calendar-month__event-multiday-title {
 		font-weight: var(--font-weight-regular);
 	}
+
+	.tribe-events-calendar-month__event-multiday-width-1 {
+		width: 100%;
```

Comment:
this should also be the default for `__event-multiday`, the modifiers should be `--width-2`, `--width-3`, and so on.

Reply:
We can get one of duration 1 if it starts in a week and it ends on the first one of the next week.

Reply:
ah you're right, my bad. either way, I think we should make this the default rather than adding a class for it.

Reply:
Yeah can do that, but ideally if we have the logic set in place to add the number dynamically to the class, we'll be adding the `-1`, unless we specifically state the conditional to be `$duration > 1`

Reply:
This will also affect the hidden ones, which have an effect on the hover. The hidden ones will not get the `--width-1` class, which is why we want this as default.